packages/utils/src/shared-schemas.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from 'zod';
+
+export const SendFeedbackSchema = z.object({
+  type: z.string(),
+  message: z.string().min(24, { message: 'Message must be at least 24 characters' }),
```

Comment:
```suggestion
  message: messageSchema,
```
Hmmm I think sometimes I don't manage to pass through to you @aaryansinha16 what is in my mind :) You are duplicating the `z.string().min(24, { message: 'Message must be at least 24 characters' })` which beats the purpose of my initial comment https://github.com/adsviewer/turboviewer/pull/545#discussion_r1777989104

Reply:
I agree, that was a silly duplication :sweat_smile: